nsxt/resource_nsxt_vpc_external_address.go

```diff
@@ -0,0 +1,109 @@
+package nsxt
+
+import (
+	"log"
+
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs/projects/vpcs/subnets"
+)
+
+func resourceNsxtVpcExternalAddress() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtVpcExternalAddressCreate,
+		Read:   resourceNsxtVpcExternalAddressRead,
+		Update: resourceNsxtVpcExternalAddressUpdate,
+		Delete: resourceNsxtVpcExternalAddressDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtParentPathResourceImporter,
+		},
+		Schema: map[string]*schema.Schema{
+			"parent_path":                getPolicyPathSchema(true, true, "port path for address binding"),
+			"allocated_external_ip_path": getPolicyPathSchema(true, false, "allocated ip path"),
+			"external_ip_address": {
```

Comment:
Did we check if this is actually computed, or can instead be used to provide an IP address not allocated from a block? (ie: a random IP selected by the user)

Reply:
Its marked as `read-only` in the spec